packages/devtools_app/lib/src/screens/memory/memory_tracker_model.dart

```diff
@@ -61,7 +62,7 @@ class TrackerCall extends Tracker {
   TrackerCall(String entry) : super._internal(name: entry);
 
   @override
-  String toString() => name;
+  String toString() => name ?? '';
```

Comment:
if entry is non nullable we should be able to null-assert here instead of providing a default value

Reply:
It is nullable and it is assigned to null in TrackerAllocation constructor.

Reply:
This can still be null-asserted in the toString method because TrackerCall takes a non-nullable parameter `entry`, which it assigns to `Tracker.name`